### PR TITLE
Fix missing fonts under Linux

### PR DESCRIPTION
**Pull request by KyleGospo**
This PR replaces Arial with Lato, and Lucidia with Northstar Mono (Modified Fira Mono Regular), removing the need to install fonts when running the game on Linux. ~~Licenses for these fonts have been included along-side the TTF.~~

Adjusts .gitattributes and .github/workflows/encoding.yml to better target localization files, this fixes a crash caused by the font definition file being encoded as UTF-16LE-BOM

Related PRs:
https://github.com/R2Northstar/Northstar/pull/314
https://github.com/R2Northstar/NorthstarWiki/pull/91

### CODE REVIEW

**Comment by KyleGospo** [comment]
Arial & Lucidia Console:
ArialLucidia

Lato & Source Code Pro:
LatoSCP

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]
(Sadly) quick Discord poll said no to `Fira code` ;-;


Consolas was a suggested alternative

**Comment by GeckoEidechse** [comment]
An issue we also noticed in testing that streaming the in-game console with `Fira Code` from a 1440p screen to a 1080p via Discord on Text readability mode, rendered the text unreadable on the 1080p screen. So it should be big enough for it to be still readable like the current version is. ^^

**Comment by KyleGospo** [comment]
> An issue we also noticed in testing that streaming the in-game console with `Fira Code` from a 1440p screen to a 1080p via Discord on Text readability mode, rendered the text unreadable on the 1080p screen. So it should be big enough for it to be still readable like the current version is. ^^

Created a custom font to solve this. Meet Fira Mono Northstar:
FiraNorthstar

File: Northstar.Client/mod/resource/fontfiletable.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"DefaultBold"				"resource/NotoSansTC-Regular.vfont"		[$TCHINESE]
+
+	"Titanfall"					"resource/Titanfall-Regular.vfont"		[!$JAPANESE && !$TCHINESE && !$RUSSIAN]
+	"Titanfall"					"resource/NotoSansJP-Regular.vfont"		[$JAPANESE]
+	"Titanfall"					"resource/NotoSansTC-Regular.vfont"		[$TCHINESE]
+	"Titanfall"					"resource/MetronicPro-SemiBold.vfont"	[$RUSSIAN]
+
+	"marlett"					"vgui/fonts/marlett.ttf"
+
+	// Everything below is DONOTSHIP / Dev only
```
Comment (GeckoEidechse) [inline Northstar.Client/mod/resource/fontfiletable.txt:18]: Uh, is this still relevant?

Comment (GeckoEidechse) [reply Northstar.Client/mod/resource/fontfiletable.txt:18]: Turns out that comment was added by Respawn in the original file inside the VPK :,)

**Review by GeckoEidechse — APPROVED** [review]
Got enough positive feedback on the updated font to consider it mergeable ^^

**Pull request merged**